app/src/main/java/org/glucosio/android/activity/AddGlucoseActivity.java

```diff
@@ -122,7 +126,7 @@ public void onNothingChosen(View labelledSpinner, AdapterView<?> adapterView) {
             GlucoseReading readingToEdit = presenter.getGlucoseReadingById(this.getEditId());
 
             String readingString;
-            if (presenter.getUnitMeasuerement().equals("mg/dL")) {
+            if (presenter.getUnitMeasuerement().equals(DomainConstants.MG_D_L)) {
```

Comment:
This should be ever `false`, since to reach the `else` statement the `presenter.getUnitMeasuerement()` cannot be equals `DomainConstants.MG_D_L`

Reply:
I don't get it